ci-operator/config/maistra/api/maistra-api-maistra-2.1.yaml

```diff
@@ -0,0 +1,36 @@
+build_root:
+  image_stream_tag:
+    name: maistra-builder
+    namespace: ci
+    tag: "2.1"
+  use_build_cache: true
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: unit
+  commands: |
+    make test \
+    GOBIN="/gobin" \
+    GOCACHE="/tmp/cache" \
+    GOMODCACHE="/tmp/cache" \
+    XDG_CACHE_HOME="/tmp/cache"
+  container:
+    from: src
+- as: gencheck
+  commands: |
+    make gen-check \
+    GOBIN="/gobin" \
+    GOCACHE="/tmp/cache" \
+    GOMODCACHE="/tmp/cache" \
+    XDG_CACHE_HOME="/tmp/cache"
+  container:
+    from: src
+zz_generated_metadata:
```

Comment:
Are these meant to be here? Or are they auto-generated?

Reply:
zz_generated_metadata is auto-generated.
container.from: src is referring to use an image with source code from a repo.

Reply:
yeah, I was referring to the `zz_generated_metadata` field. So, this whole file is auto-generated then?

Reply:
we can manually update the zz_generated_metadata field. But when we run a required `make update` command from this openshift/release repo before pushing a PR change, it will be overwritten to match the file name convention.
And it is used for defining which github repo and branch the job is watching.